connector/src/main/java/org/geysermc/connector/configuration/GeyserJacksonConfiguration.java

```diff
@@ -29,6 +29,8 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Getter;
 import lombok.Setter;
+
```

Comment:
We've had this before where it's a Visual Studio thing, but typically we have all non-Java-namespace imports on one block with no whitespace.